src/components/SearchFiltersPanel/SearchFiltersPanel.js

```diff
@@ -101,7 +104,7 @@ class SearchFiltersPanelComponent extends Component {
       // if no option is passed, clear the selection for the filter
       const currentQueryParams = option
         ? { ...mergedQueryParams, [urlParam]: option }
-        : omit(mergedQueryParams, urlParam);
+        : { ...mergedQueryParams, [urlParam]: null };
```

Comment:
Don't remove omit. (I assume that this will add URL parameter `category=null&amenities=null`)

Reply:
Nope, this was a bug fix. Clearing the filter didn't work without this. It removes the param from the URL correctly.